package.json

```diff
@@ -5,6 +5,7 @@
   "main": "app.js",
   "scripts": {
     "start": "env $(sed 's/ # .*//' .env) supervisor app",
+    "start:indexer": "env $(sed 's/ # .*//' .env) supervisor indexer",
```

Comment:
Can we change this to `node indexer.js`? We don't need to watch the codebase or parse the `.env` file (which won't exist anyway). If the process does die then I think we want the container to exit.

For reference the `docker:start` command is what the ECS service currently uses as its entrypoint.

Reply:
If we don't need to reference `supervisor` should we just do without a `package.json` script all together? The task definition could just include `node indexer.js`?

Reply:
Yup but it provides a convenient alias for the task definition to reference so that it doesn't need to be updated if we refactor something in this codebase. I feel like this repo should be providing the interface to start the indexer service and external consumers shouldn't need knowledge of the module structure to run it. So I'd prefer to have it even if it's functionally equivalent at the moment.

Reply:
Good points, we probably still need `.env` parsing + `supervisor` for local dev so i've kept `start:indexer` the same and added `docker:start:indexer` which is just `node indexer.js`